deployment/data-streams/changeset/verifier-proxy/v0.5.0/call_unset_verifier_test.go

```diff
@@ -0,0 +1,148 @@
+package v0_5_0
+
+import (
+	"encoding/hex"
+	"testing"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+
+	v0_5_0 "github.com/smartcontractkit/chainlink/deployment/data-streams/changeset/verifier/v0.5.0"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	commonChangesets "github.com/smartcontractkit/chainlink/deployment/common/changeset"
+	"github.com/smartcontractkit/chainlink/deployment/data-streams/changeset/testutil"
+	"github.com/smartcontractkit/chainlink/deployment/data-streams/changeset/types"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/llo-feeds/generated/verifier_proxy_v0_5_0"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/llo-feeds/generated/verifier_v0_5_0"
+)
+
+func TestUnsetVerifier(t *testing.T) {
+	e := testutil.NewMemoryEnv(t, true)
+
+	chainSelector := e.AllChainSelectors()[0]
+	e, err := commonChangesets.Apply(t, e, nil,
+		commonChangesets.Configure(
+			DeployVerifierProxyChangeset,
+			DeployVerifierProxyConfig{
+				ChainsToDeploy: []uint64{chainSelector},
+				Version:        *semver.MustParse("0.5.0"),
+			},
+		),
+	)
+	require.NoError(t, err)
+
+	// Ensure the VerifierProxy was deployed
+	verifierProxyAddrHex, err := deployment.SearchAddressBook(e.ExistingAddresses, chainSelector, types.VerifierProxy)
+	require.NoError(t, err)
+	verifierProxyAddr := common.HexToAddress(verifierProxyAddrHex)
+
+	// Deploy Verifier
+	e, err = commonChangesets.Apply(t, e, nil,
+		commonChangesets.Configure(
+			v0_5_0.DeployVerifierChangeset,
+			v0_5_0.DeployVerifierConfig{
+				ChainsToDeploy:       []uint64{chainSelector},
+				VerifierProxyAddress: verifierProxyAddr,
+				Version:              *semver.MustParse("0.5.0"),
+			},
+		),
+	)
+	require.NoError(t, err)
+
+	// Ensure the Verifier was deployed
+	verifierAddrHex, err := deployment.SearchAddressBook(e.ExistingAddresses, chainSelector, types.Verifier)
+	require.NoError(t, err)
+	verifierAddr := common.HexToAddress(verifierAddrHex)
+
+	chain := e.Chains[chainSelector]
+
+	verifier, err := verifier_v0_5_0.NewVerifier(verifierAddr, e.Chains[chainSelector].Client)
+	require.NoError(t, err)
+	require.NotNil(t, verifier)
+
+	// Initialize the verifier
+	e, err = commonChangesets.Apply(t, e, nil,
+		commonChangesets.Configure(
+			InitializeVerifierChangeset,
+			VerifierProxyInitializeVerifierConfig{
+				ConfigPerChain: map[uint64]InitializeVerifierConfig{
+					testutil.TestChain.Selector: {
+						VerifierAddress: verifierAddr,
+						ContractAddress: verifierProxyAddr,
+					},
+				},
+			},
+		),
+	)
+	require.NoError(t, err)
```

Comment:
I was wandering if we should create some test utils for 
Deploy VerifierProxy, Verifier and InitializeVerifier

Reply:
Did it here https://github.com/smartcontractkit/chainlink/blob/feat/verifier/deployment/data-streams/changeset/verifier/v0_5_0/test_helpers.go#L20

Reply:
great thank you